### ISSUE TITLE
Github node Get file content is not saving json to binary data starting from n8n ver 0.193.x

### ISSUE THREAD

**Issue by ed-parsadanyan** [n8n-io/n8n#4015]
**Describe the bug**
I was using an example of n8n workflow that pushes all workflows to the Github. It was working up until 0.192.2 version but fails on 0.193.x The Github node that reads the file content fails to save it into binary data.

```
Stack
TypeError: Cannot read properties of undefined (reading 'json')
    at Object.execute (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Github/Github.node.js:1796:67)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Workflow.runNode (/usr/local/lib/node_modules/n8n/node_modules/n8n-workflow/dist/src/Workflow.js:598:28)
    at async /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/WorkflowExecute.js:557:53
```
When the tickbox "As Binary Property" is switched off then the node works fine.

**To Reproduce**

1. Set up Github node
2. Select Resource = File, Operation = Get, provide all the details (repo owner, repo name, file path)
3. Switch on "As Binary Property"
4. Click "Run" and get the Cannot read properties of undefined (reading 'json') error

**Expected behavior**
Save the content of the file in a data binary element.

**Environment (please complete the following information):**

- OS: Ubuntu Linux 22.04
- n8n Version 0.193.3
- Node.js Version NA - Docker container
- Database system postgresql
- Operation mode main

**Additional context**
NA

**Comment by Joffcom** [comment]
Hey @ed-parsadanyan,

Can you share the workflow you are using?

**Comment by ed-parsadanyan** [comment]
@Joffcom
https://gist.github.com/ed-parsadanyan/d9215c7b271117357adb7a0b4e697105

The first Github node was originally set to skip errors, but they are still there.
Usually when the repo file really doesn't exist it throws 404 error, which is expected. But when the file exists, the node cannot save it into binary data for some reason.

**Comment by Joffcom** [comment]
Hey @ed-parsadanyan,

Perfect I will give that a go either over the weekend or on Monday morning.

Looking at the workflow json data it looks like it might be the back up workflow I made that uses the command line tool to export the data which is handy as that is the one I use on my own instances.

I will be back once I have checked it out.

**Comment by Joffcom** [comment]
Hey @ed-parsadanyan,

Just managed to reproduce the issue, It looks like I didn't pick up the issue sooner as I don't use the binary data option for my backups but I have just popped in a PR which will fix this in an upcoming release. Thanks for letting us know about this one 👍🏻

**Comment by janober** [comment]
Fix got released with `n8n@0.193.4`
